app/view_models/paye/TaxYearFormatting.scala

```diff
@@ -20,7 +20,7 @@ trait TaxYearFormatting {
 
   val taxYear: Int
 
-  def taxYearStart = taxYear.toString
-  def taxYearEnd = (taxYear + 1).toString
-
+  def taxYearFrom: String = (taxYear - 1).toString
+  def taxYearTo: String = taxYear.toString
+  def taxYearInterval: String = (taxYear - 1).toString + "-" + taxYear.toString.substring(2)
```

Comment:
@jpatton638 this might mess up with the "year to year" changes you had made, we'll see!